app/furniture/marketplace/marketplaces_controller.rb

```diff
@@ -9,5 +9,17 @@ def show
     def edit
       authorize(marketplace)
     end
+
+    def update
+      if authorize(marketplace).update(marketplace_params)
+        redirect_to marketplace.location(child: :notification_methods), notice: t(".success", name: marketplace.room.name)
```

Comment:
🤔 Actually not seeing any `notice` on screen....

Reply:
In rails, the `flash` method sets a cookie; which is included in the response payload of the request response cycle. 

Because every request in the `Marketplace` is running through `Turbo`,  the only bits that are re-rendered are the elements within the turbo frame.

If you want to see data in `flash[:notice]`, you'll need to add them to something within that frame you could [the `marketplace` layout](https://github.com/zinc-collective/convene/blob/30d64fe782444d2b97c83cfe78e54cc644619e61/app/furniture/layouts/marketplace.html.erb#L3-L10) to include the flash messages.